server/src/main/java/org/elasticsearch/index/query/QueryShardContext.java

```diff
@@ -231,6 +231,10 @@ public MappedFieldType fieldMapper(String name) {
         return failIfFieldMappingNotFound(name, mapperService.fieldType(name));
     }
 
+    public MappedFieldType fieldType(String name) {
```

Comment:
Can we put some javadocs on here, specifically around the fact that it may return `null`?

Reply:
Too fast! Can you add a note about `null`?